run-maelstrom

```diff
@@ -2,4 +2,5 @@
 
 # A small wrapper script for invoking the Maelstrom jar, with arguments.
 
-java -jar target/maelstrom-0.2.1-SNAPSHOT-standalone.jar "$@"
+#java -jar target/maelstrom-0.2.1-SNAPSHOT-standalone.jar "$@"
+lein run "$@"
```

Comment:
Oh--yeah, I mean, there's... not really much need for this script at all, right? It's actually more characters to type `./maelstrom test` rather than `lein run test`. ;-)